scripts/west_commands/runners/bossac.py

```diff
@@ -37,35 +42,17 @@ def capabilities(cls):
     def do_add_parser(cls, parser):
         parser.add_argument('--bossac', default='bossac',
                             help='path to bossac, default is bossac')
-        parser.add_argument('--offset', default=None,
-                            help='start erase/write/read/verify operation '
-                                 'at flash OFFSET; OFFSET must be aligned '
-                                 ' to a flash page boundary')
```

Comment:
I prefer have options at command line but I don't see why keep this. It not help and can create confusion.

1. In a baremetal environment, for instance, the offset must have a entry on the linker, otherwise it will not work. If we keep this option, the "linker" must be defined anyway on devicetree. As far I could understand it seams to be redundant.
2. In a multi image environment (secure/non-secure) there should be at least 2 offsets. west can get the offset from `.config` file and flash one images at time.

Do you see a scenario that may require --offset when invoke west at cmd line?

Reply:
> Do you see a scenario that may require --offset when invoke west at cmd line?

So you're saying the user should **never** provide this option, and it was always wrong to have it? If so, OK.

Reply:
Yes, exactly!